UPGRADE.md

```diff
@@ -0,0 +1,12 @@
+UPGRADE FROM 1.0.0-beta.1 to 1.0.0-beta.2
+================================
+
+# Table of Contents
+
+- [CoreBundle](#corebundle)
+
+### CoreBundle
+
+* ImageUniqueCode and ImageUniqueCodeValidator was deleted and replaced by WithinCollectionUniqueCodeValidator, WithinCollectionUniqueCode form ResourceBundle.
```

Comment:
Let's write the class names using monospace font. Also "was" -> "were", "form" -> "from" (and the order of classes is different 🌮 ).